Headers/SPIFFSFileHandler/SPIFFSFileHandler.cpp

```diff
@@ -0,0 +1,88 @@
+/*
+   This  library is  created with necessary functions to use SPIFFS in ESP8266.
+*/
+
+#include <string.h>
+#include <ESP8266WiFi.h>
+#include "FS.h"
+#include "SPIFFSFileHandler.h"
+
+*
```

Comment:
Hope you have tested this code before uploading it... but I doubt whether it will give errors here as the commenting is not proper.

Reply:
ok